spec/paperclip/storage/s3_spec.rb

```diff
@@ -3,17 +3,24 @@
 
 describe Paperclip::Storage::S3 do
   before do
-    AWS.stub!
+    if defined?(::Aws)
+      Aws.config[:stub_responses] = true
+    else
+      AWS.stub!
+    end
+  end
+
+  def aws2_add_region
+    defined?(::Aws) ? { s3_region: 'us-east-1' } : {}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.